addon/components/pix-multi-select.js

```diff
@@ -17,11 +23,17 @@ export default class PixMultiSelect extends Component {
       option.checked = this._selected.includes(option.value);
     });
 
+    if(this.searchData) {
+     return defaultSelected.filter(list => 
+       list.label.includes(this.searchData)
+     );
+    }
```

Comment:
**suggestion (potentiellement bloquant)**
Le filtre semble sensible à la casse (si on met en majuscule ou pas on voudrait le même résultat non ?).
Le moyen le plus simple semble de mettre tout en majuscule (https://stackoverflow.com/questions/2140627/how-to-do-case-insensitive-string-comparison), ce qui donnerait quelque chose du genre :

```suggestion
    upperCaseList = list.map(element => element.label.toUpperCase())
    if(this.searchData) {
     return defaultSelected.filter(list => 
       upperCaseList.label.includes(this.searchData.toUpperCase())
     );
    }
```

Reply:
dans les propositions que j'ai faite à Xavier, il y a la possibilité d'avoir un paramètre `strictSearch` pour spécifier si oui ou non on prend en compte la casse, les accents et les signes diacritiques.

Reply:
j'avais fait un truc insensible a la casse et au accent. on la retiré. mais au final on le remet. c'est dans un futur commit qui arrive. 

avec la possibilité de le rendre sensible ou non .